tests/e2e/tests/memcache_test.py

```diff
@@ -0,0 +1,39 @@
+"""Tests for the memcache API.
+
+This test module is incomplete but is a starter.
```

Comment:
This sort of comment isn't necessary. You can always file github bugs to follow-up.